modules/compression/package.json

```diff
@@ -28,8 +28,11 @@
     "README.md"
   ],
   "scripts": {
-    "pre-build": "npm run build-bundle && npm run build-bundle -- --env.dev",
-    "build-bundle": "webpack --display=minimal --config ../../scripts/webpack/bundle.js"
+    "pre-build": "npm run build-bundle && npm run build-workers && npm run build-bundle -- --env.dev",
```

Comment:
Nit: use yarn instead of npm run?

Reply:
Yes. We'd need to change in many places, so separate PR.